### PR TITLE
Only get changelog authors from last tag

### PR DESCRIPTION
**Pull request by Zlopez**
Only small update for get-authors.py script.

This will check git log only from last tag.

### CODE REVIEW

File: news/get-authors.py [hunk 1]
```diff
@@ -2,8 +2,10 @@
 
 from subprocess import check_output
 
+last_tag = check_output(["git", "describe", "--abbrev=0"])
 authors = {}
-output = check_output(["git", "log", "--format=%ae\t%an"], universal_newlines=True)
+log_range = last_tag.decode("utf-8").strip() + "..HEAD"
```
Comment (abompard) [inline news/get-authors.py]: You could also use `universal_newlines=True` to convert the bytes to a string. This is what is used by the other `check_output` call.

Comment (Zlopez) [reply news/get-authors.py]: Thanks for hint. I will do this.

Comment (Zlopez) [reply news/get-authors.py]: Still need to strip for '\n'.

**Comment by Zlopez** [comment]
Rebased

**Comment by abompard** [comment]
Thanks. Please run `black` on the file, that's the only failing test now :-)

**Comment by Zlopez** [comment]
Didn't knew, that something like `black` exists. Looks really nice.

**Comment by abompard** [comment]
Yeah it's really cool! There are developer instructions in our docs if you want to use it, but's basically all on their website. There are editor integrations too, I use it with vim & VSCode and it works great.

**Pull request merged**